src/com/apmath/loans/application/v1/actions/ListLoans.kt

```diff
@@ -0,0 +1,40 @@
+package com.apmath.loans.application.v1.actions
+
+import com.apmath.loans.application.v1.models.Mixed
+import com.apmath.loans.application.v1.models.toMixedId
+import com.apmath.loans.application.v1.respondError
+import com.apmath.loans.application.v1.validators.MixedBuilder
+import com.apmath.loans.domain.services.LoanServiceInterface
+import com.apmath.validation.simple.NullableValidator
+import io.ktor.application.ApplicationCall
+import io.ktor.response.respond
+
+suspend fun ApplicationCall.v1ListLoans(loanService: LoanServiceInterface) {
+    val mixed = Mixed(
```

Comment:
@malinink suggested moving from model to vars
like [here?](https://github.com/apmath-web/loans/pull/28/commits/0eb8b1bebe77931c75d2151530da926009581289)